doc/doc_guide.rst

```diff
@@ -0,0 +1,201 @@
+| @copybrief            | Copy the brief        | ``@copybrief slice``  |
+|                       | description from the  |                       |
+|                       | object specified      |                       |
++-----------------------+-----------------------+-----------------------+
+| @copydetails          | Copy the detailed     | ``@copydetails dot``  |
+|                       | documentation from    |                       |
+|                       | the object specified  |                       |
++-----------------------+-----------------------+-----------------------+
+| @note                 | Add a note message    | ``@note this is named |
+|                       | where the text will   |  after an equivalent  |
+|                       | be highlighted        | function in PyTorch`` |
++-----------------------+-----------------------+-----------------------+
+| @warning              | Add a warning message | ``@warning            |
+|                       | where the text will   | not implemented``     |
+|                       | be highlighted        |                       |
++-----------------------+-----------------------+-----------------------+
+| @b                    | Display a single word | ``@b bold``           |
+|                       | using a bold font     |                       |
++-----------------------+-----------------------+-----------------------+
+| @c                    | Display a single word | ``@c void``           |
+|                       | using a typewriter    |                       |
+|                       | font                  |                       |
++-----------------------+-----------------------+-----------------------+
+| @p                    | Display a single word | ``@p transA``         |
+|                       | using a typewriter    |                       |
+|                       | font. Equivalent to   |                       |
+|                       | ``@c``                |                       |
++-----------------------+-----------------------+-----------------------+
+| @em                   | Display a single word | ``@em x``             |
+|                       | in italics.           |                       |
++-----------------------+-----------------------+-----------------------+
+
+.. note::
+
+    Not all Doxygen special commands are supported in Exhale, e.g., `grouping`_.
+    Some commands like `@name`_ could lead to errors when parsing overloaded functions.
+    To free yourself from debugging the Doxygen comments for hours, we recommend you only using the
+    above commands.
+
+| **How to include math formulas in Doxygen?**
+| Doxygen supports LaTeX math formulas in the documation. To include an inline formula that appears
+  in the running text, we need wrap it by a pair of ``@f$`` commands, for example:
+
+.. code:: none
+
+    Default is no smoothing, @f$\alpha = 0 @f$.
+
+This will result in: Default is no smoothing, |formula1|.
```

Comment:
Sphinx doesn't generate an image with math formula here for me and just keeps `|formula1|`. Is it supposed to do so?

Reply:
The image is missing. Now it should show as a inline image. Thanks for pointing out.